bio/hmmer/jackhmmer/wrapper.py

```
@@ -0,0 +1,38 @@
+from snakemake.shell import shell
+
+
+output_parameters = [
+    ("--tblout", "tblout"),
+    ("--domtblout", "domtblout"),
+    ("-A", "alignment_hits"),
+]
+
+
+# Output must be set even if its not desired.  Default to '/dev/null'
+out_cmd = f" -o {snakemake.output.get('outfile', '/dev/null')}",
```

Ruff (0.8.2)

Undefined name `snakemake`

(F821)